UKHomeOffice/cerberus-service
# COP-9871: Update test to verify tasks refreshed every 3mins

## Description
Update test to verify tasks refreshed every 3mins

## To Test
npm run cypress:runner

run the test `Should verify refresh task list page` from spec `task-management.spec.js`

## Developer Checklist

\* Required
